partners.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="/style.css">
+  <link rel="stylesheet" href="/index.css">
+  <title>About-page</title>
+</head>
+<body>
+  <header class="header-container" >
+    <div class="first-section-nav">
+      <ul class="section-nav-list">
+        <li ><img class="section-nav-item" src='/assets/facebook.png' alt="logo"></li>
+        <li ><img class="section-nav-item" src='/assets/twitter.png' alt="logo"></li>
+        <li>French</li>
+        <li>English</li>
+        <li>Log out</li>
+      </ul>
+    </div>
+    <div class="header">
+      <button class="moile-menu-hamburger">
+        <span class="hamburger"></span>
+      </button>
+      <h2 class="logo-uncc"> <a href="/index.html">UNCC</a></h2>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-item"><a href="/partners.html">About</a></li>
```

Comment:
- [ ] The _**About Page**_ should have a link that directs visitors of this app to the main page. Kindly change this to the link to the `Home Page`. 👍